test/php/gutenberg-coding-standards/Gutenberg/ruleset.xml

```diff
@@ -6,5 +6,6 @@
 	<rule ref="Gutenberg.CodeAnalysis.GuardedFunctionAndClassNames"/>
 	<rule ref="Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses"/>
 	<rule ref="Gutenberg.NamingConventions.ValidBlockLibraryFunctionName"/>
+	<rule ref="Gutenberg.Commenting.FunctionCommentSinceTag" />
```

Comment:
I wonder if the sniff should be renamed as now it is not limited anymore to checking the presence of the `@since` tag in functions? I'm not sure how this sniff is used, so I don't know if there are any implications to changing its name.